src/containers/header/index.css

```diff
@@ -0,0 +1,68 @@
+#header{
+	overflow: hidden;
+	padding: 15px;
+}
+
+#buttonChat{
+	height: 54px;
+	width: 133px;
+	border-radius: 5px;
+	background-color: #0DABAF;	
+	box-shadow: 0 2px 6px 0 rgba(0,0,0,0.25);
+	float: left;
+	margin: 10px;
+	font-family: sans-serif;
+	color: white;
+	font-size: 17px;
+	border: none;
+}
+
+#buttonGroup{
+	height: 54px;
+	width: 146px;
+	border-radius: 5px;
+	background-color: #0DABAF;	
+	box-shadow: 0 2px 6px 0 rgba(0,0,0,0.25);
+	float: left;
+	margin-left: 5px;
+	margin-top: 10px;
+	font-family: sans-serif;
+	color: white;
+	font-size: 17px;
+	border: none;
+}
+
+#accountSettings{
+	float: right;
+	padding-top: 10px;
+}
+
+#chatName{
+	font-family: sans-serif;
+	color: white;
+	float: left;
+	margin-right: 1px;
+}
+
+#buttonProfile{
+	border-radius: 50%;
+	width: 50px;
+	height: 50px;
+	background-color: white;
+	margin-left: 5px;
+	float: left;
+	border: none;
+	padding: 0px;
+}
+
+#buttonAbout{
+	border-radius: 50%;

```

Comment:
why buttonAbout and buttonProfile? why not just a button?

Reply:
they have different values for "margin-left" but I will make them reference one. do you think I should do the same for "new chat" and "new group" buttons as well? Never mind Kimothy gave it an "okay"